testing/bdd/features/validation/sequence_id_num.feature

```diff
@@ -1,6 +1,11 @@
 @validation @syntax @sequence @xsd
 Feature: Sequence ID and Sequence Number
-  Both are mandatory parameters and the sequence number has to be a number (no letters)
+  Both are mandatory parameters and the sequence number has to be a number (no letters).
+  Moreover two documents with the same sequence identifier shall have different sequence numbers.
```

Comment:
So sequence identifier and sequence number are the most obvious attributes to compare documents, but is it enough ? Because if it is explicitly authorized to send the exact same document twice (it might happen actually, in case of carriage defect for example,) then we need to also use other attributes to compare documents. @kozmaz87 @tairt @nigelmegitt 

For example the document would have to use the exact same timings and to have the exact same content.